python/tests/xir/test_integration.py

```diff
@@ -55,6 +55,8 @@ def parse_script(circuit: str) -> XIRProgram:
 g_one(pi) | [0, 1];
 g_two | [2];
 g_three(1, 3.3) | [2];
+adjoint g_four(1.23) | [2];
+ctrl[0, 2] g_five(3.21) | [1];
```

Comment:
Will control and adjoint still work with custom gates like `h`?

e.g. `adj h(0.2) | [0, 1, 2];`

Reply:
Yes, it does!